docs/make.jl

```diff
@@ -6,15 +6,26 @@ makedocs(
     sitename = "Caesar.jl",
     pages = Any[
         "Home" => "index.md",
-        "Getting Started" => "getting_started.md",
-        "Examples" => "examples.md",
-        "Entry Tutorial: ContinuousScalar" => "tutorialcontinuousscalar.md",
-        "Entry Tutorial: Hexagonal 2D SLAM" => "tut_hexagonal2d.md",
-        "Entry Tutorial: Singular Ranges-only SLAM" => "tut_slamedonut.md",
-        "Moderate Tutorial: Defining Factors" => "definingfactors.md",
-        "Arena Visualization" => "arena_visualizations.md",
-        "Offloading to Server" => "database_interactions.md",
-        "Functions" => "func_ref.md"
+        "Getting Started" => [
+            "Installation" => "installation_environment.md"
+        ],
+        "Concepts" => [
+            "Caesar Concepts" => "concepts/concepts.md",
+            "Building Factor Graphs" => "concepts/building_graphs.md",
+            "Adding New Variables and Factors" => "concepts/adding_variables_factors.md",
+            "Arena Visualization" => "concepts/arena_visualizations.md",
+            "Using Caesar's Multi-Language Support" => "concepts/zmq.md",
```

Comment:
maybe `multilang` instead of `zmq` wherever possible?

Reply:
+1

multilang (using ZMQ)

Reply:
Done! Thanks :gear: